src/main/java/lu/oop/server/api/controllers/MessageController.java

```diff
@@ -37,17 +39,7 @@ public int getReceiverId(){
         this.messageService = MessageService;
     }
 
-    @GetMapping("/{id}")
-    public ResponseEntity<IMessageModel> getMessageById(@PathVariable Long id) {
-        Optional<IMessageModel> mbyMessage = messageService.getById(id);
-        if(mbyMessage.isEmpty()) {
-            return ResponseEntity.notFound().build();
-        }
-        IMessageModel message = mbyMessage.get();
-        return ResponseEntity.ok(message);
-    }
-
-    @PostMapping(value = "/")
+    @PostMapping("/")
```

Comment:
Didn't work for me. I had to change it to @PostMapping() to get it to work.

Reply:
Wait change what? Dont get this one